recipes/zserio/all/zserio_compiler.cmake

```diff
@@ -0,0 +1,41 @@
+function(zserio_generate_cpp GENERATED_SOURCES_OUT)
```

Comment:
I'm curious if the project does not provide this cmake macro already? It could be copied to the package folder directly.

Reply:
Unfortunately we don't have the cmake function yet in the current zserio release. I would like to add it to the next 2.14 release and maybe then we could just remove the 2.13 from the conan repository.